flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/options/HBaseWriteOptions.java

```diff
@@ -100,19 +98,15 @@ public static Builder builder() {
 	 */
 	public static class Builder {
 
-		// default is 2mb which is defined in hbase
 		private long bufferFlushMaxSizeInBytes = ConnectionConfiguration.WRITE_BUFFER_SIZE_DEFAULT;
-		private long bufferFlushMaxRows = -1;
-		private long bufferFlushIntervalMillis = -1;
+		private long bufferFlushMaxRows = 0;
```

Comment:
I would prefer `-1` as the default value, which means the value is not set. `0` seems to mean "never flush" .

Reply:
I have tried, but Duration and MemorySize ConfigOptions doesn't allow negative values.